### PR TITLE
Enhancing OpenStackComputeQuotaPlugin tests according to the new specifications

### PR DESCRIPTION
**Pull request by llvieira**

### CODE REVIEW

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java [hunk 1]
```diff
@@ -40,41 +40,56 @@ public void setUp() {
...
 	    Mockito.doThrow(FogbowManagerException.class).when(this.plugin).getUserQuota(this.token);
+
+	    // verify
 	    Mockito.verify(this.plugin).getUserQuota(Mockito.eq(this.token));
 
 	}
-	
-	@Test (expected = FogbowManagerException.class)
+
+    // test case: Verify getUserQuota() when it raises UnexpectedException.
```
Comment (ordansantos) [inline src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java]: This method is actually doing nothing, please, run code coverage and notice this method breaks in the very first line.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java [hunk 2]
```diff
@@ -40,41 +40,56 @@ public void setUp() {
...
-        Assert.assertEquals(allocationAvailable.getvCPU(), this.plugin.getUserQuota(this.token).getAvailableQuota().getvCPU());
-        Assert.assertEquals(allocationAvailable.getRam(), this.plugin.getUserQuota(this.token).getAvailableQuota().getRam());
-        Assert.assertEquals(allocationAvailable.getInstances(), this.plugin.getUserQuota(this.token).getAvailableQuota().getInstances());
+        Assert.assertEquals(allocationAvailable.getvCPU(), availableQuota.getvCPU());
+        Assert.assertEquals(allocationAvailable.getRam(), availableQuota.getRam());
+        Assert.assertEquals(allocationAvailable.getInstances(), availableQuota.getInstances());
     }
-	
+
+    // test case: Verify getUserQuota() when it raises FogbowManagerException.
```
Comment (ordansantos) [inline src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java]: This method is actually doing nothing, please, run code coverage and notice this method breaks in the very first line.

Comment (ordansantos) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java]: In addition to that, it seems this method is trying to test a mocked method, I think we should replace this whole test with another one.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java [hunk 3]
```diff
@@ -40,41 +40,56 @@ public void setUp() {
 		token.setAttributes(attributes);
 		token.setAccessId("");
 	}
-	
+
+	// test case: Tests if getTotalQuota(), getUsedQuota() and getAvailableQuota() returns the right quotas.
     @Test
     public void getUserQuotaTest() throws FogbowManagerException, UnexpectedException {
+	    // set up
         ComputeAllocation allocationTotal = new ComputeAllocation(64, 46080, 20);
```
Comment (ordansantos) [inline src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java]: Where these magic numbers come from? Were they calculated by hand? We should think in a better way to generate this type of data.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java [hunk 4]
```diff
@@ -40,41 +40,56 @@ public void setUp() {
 		token.setAttributes(attributes);
 		token.setAccessId("");
 	}
-	
+
+	// test case: Tests if getTotalQuota(), getUsedQuota() and getAvailableQuota() returns the right quotas.
```
Comment (ordansantos) [inline src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java]: returns the right quotas from where?

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java (hunk 2 above)
Comment (llvieira) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java]: I agree, this whole test make no sense. I'm going to take it away.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java (hunk 1 above)
Comment (llvieira) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java]: Another one which does not make sense.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java (hunk 3 above)
Comment (llvieira) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java]: Now I reproduced the method where it is generated.

**Pull request merged**